docs/pywbemcli/commands.rst

```diff
@@ -1556,115 +1556,67 @@ command.
-Time`` as follows.
-
-.. code-block:: text
-
-    pywbemcli>statistics show
-    Statistics: Time(Seconds); Times/lengths: (Avg/Min/Max), single value if all same.
-       Op    Exc  Op Time(S)         Server Time(S)     RequestLen     ReplyLen        Operation
-      Cnt    Cnt  Avg/Min/Max        Avg/Min/Max        Avg/Min/Max    Avg/Min/Max
-    -----  -----  -----------------  -----------------  -------------  --------------  ----------------------
-        2      0  0.045/0.041/0.049  0.000/inf/0.000    332            63309           EnumerateClassNames
-        2      1  0.026/0.005/0.047  0.000/inf/0.000    333/321/345    6225/298/12152  EnumerateInstanceNames
-        5      2  0.014/0.004/0.033  0.000/0.001/0.001  550/536/560    1728/298/3838   OpenEnumerateInstances
-        1      0  0.013              0.001              3483           224             ModifyInstance
-        2      0  0.010/0.010/0.010  0.000/0.001/0.001  345            3504/3504/3505  EnumerateInstances
-        1      0  0.008              0.001              822            2976            GetInstance
-        2      0  0.006/0.006/0.006  0.000/inf/0.000    542            4187            GetClass
-
-Statistics maintained in the server are only displayed if the server is
-gathering statistics and only with the pywbemcli command ``statistics
-server-show``. Pywbemcli does not provide any mechanism to reset these
-statistics values and simply reports the statistics based on the data received
-from the WBEM server.
-
-In the interactive mode pywbemcli statistics are discarded each time a new WBEM
-server is selected but the statistics display mode (--timestats) is maintained.
+1.  Pywbemcli gathers and maintains statistics on CIM-XML operations it executes
+    against a WBEM server (or mock environment). These client statistics can be
+    displayed either after each pywbemcli command if the ``--timestats`` / ``-T``
+    general option is used, or in interactive mode with the ``statistics show``
+    command.
+
+    For mock environments, artificial operations on the MOF compile time
+    needed for setting up the mock respository are included in the statistics.
+
+2.  WBEM servers may support two capabilities for managing statistics on WBEM
+    operations:
+
+    a. Each CIM-XML response from the WBEM server may include an extra header
+       field ``WBEMServerResponseTime`` with the server response time for that
+       operation. Pywbemcli puts those server response times into the client
+       statistics it maintains. The inclusion of the server response time
+       can be enabled and disabled with the ``statistics server-on`` and
+       ``statistics server-off`` commands.
+
+    b. Statistical information on operation execution in the WBEM server and
+       its providers may be gathered and maintained and by the WBEM server.
+       This server statistics is completely independent of the client statistics
```

Comment:
Probably should be These server statistics are

Reply:
DONE